pkg/lib/strings/operations.go

```diff
@@ -226,3 +227,25 @@ func PluralCustom(singular string, plural string, count interface{}) string {
 	}
 	return plural
 }
+
+// RemoveDuplicates returns a filtered string slice without repeated entries.
+// The ignoreRegex parameter can optionally be used to ignore repeated patters in each slice entry.
+func RemoveDuplicates(strs []string, ignoreRegex *regexp.Regexp) []string {
+	var result []string
+	counter := map[string]int64{}
```

Comment:
`"github.com/cortexlabs/cortex/pkg/lib/sets/strset"` can be used here.

Reply:
what are the wins here?

Reply:
I've looked at the code, and I don't see how I can use the library to replace the counter

Reply:
We can not use strset if we want to reduce dependencies in the lib